content/en/serverless/distributed_tracing/_index.md

```diff
@@ -126,11 +116,13 @@ If `dd-trace` is installed on your hosts with the Datadog Agent, and your server
 
 Datadog's [AWS X-Ray integration][2] only provides traces for Lambda functions. See the [Datadog APM documentation][16] to learn more about tracing in container or host-based environments.
 
-## Enable Datadog APM
+## Profiling your Lambda Functions (Public Beta)
 
-{{< img src="tracing/live_search/livesearchmain.mp4" alt="Live Search" video=true >}}
+<div class="alert alert-info">During the beta period, profiling is available at no additional cost.</div>
 
-The Datadog Python, Node.js, Ruby, Go, Java, and .NET tracing libraries support distributed tracing for AWS Lambda. To enable tracing on your functions, follow [the installation instructions][5].
+Datadog's [Continuous Profiler][27] is now available in beta for Python in version 4.62.0 and layer version 62 and above. This optional feature is enabled by setting the `DD_PROFILING_ENABLED` environment variable to true. 
+
+The Continuous Profiler works by spawning a thread which periodically wakes up and takes a snapshot of the CPU and Heap of all running python code. This can include the profiler itself. If you want the Profiler to ignore itself, set `DD_PROFILING_IGNORE_PROFILER` to true.
```

Comment:
```suggestion
The Continuous Profiler works by spawning a thread that periodically wakes up and takes a snapshot of the CPU and heap of all running Python code. This can include the profiler itself. If you want the profiler to ignore itself, set `DD_PROFILING_IGNORE_PROFILER` to `true`.
```